.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: 💬 Question
+    url: https://libero.pub/join-slack
+    about: Have a question? Ask it on the Libero Community Slack.
```

Comment:
Not done this before, and hard to find what it would look like (eventually found https://github.com/haproxy/haproxy/issues/new/choose).